lib/reporter/worker.rb

```diff
@@ -86,6 +86,7 @@ def extract(objects) # rubocop:disable  Metrics/MethodLength
 
         entity_class.new(
           instance,
+          current_user: @author.id,
```

Comment:
should it not be  `@author`  instead of `@author.id`?

Reply:
you are right. I fixed the typo